app/src/main/java/org/wikipedia/navtab/NavTabLayout.java

```diff
@@ -23,10 +25,20 @@ public NavTabLayout(Context context, AttributeSet attrs, int defStyleAttr) {
         setTabViews();
     }
 
-    private void setTabViews() {
+    public void setTabViews() {
         for (int i = 0; i < NavTab.size(); i++) {
             NavTab navTab = NavTab.of(i);
-            getMenu().add(Menu.NONE, i, i, navTab.text()).setIcon(navTab.icon());
+            if (AccountUtil.isLoggedIn()) {
+                getMenu().add(Menu.NONE, i, i, navTab.text()).setIcon(navTab.icon());
+            } else {
+                if (!(NavTab.SUGGESTED_EDITS == navTab)) {
+                    getMenu().add(Menu.NONE, i, i, navTab.text()).setIcon(navTab.icon());
+                }
+            }
         }
     }
+
+    public void clear() {
+        getMenu().clear();
```

Comment:
Not sure but maybe we can add `getMenu().clear()` to the beginningg of `setTabViews()`.